### PR TITLE
Covcov

### PR DESCRIPTION
**Pull request by Hellowlol**

### CODE REVIEW

File: plexapi/client.py [hunk 1]
```diff
@@ -444,7 +444,7 @@ def playMedia(self, media, offset=0, **params):
         if not self.product != 'OpenPHT':
             try:
                 self.sendCommand('timeline/subscribe', port=server_url[1].strip('/'), protocol='http')
-            except:
+            except:  # noqa: E722
```
Comment (pkkid) [inline plexapi/client.py]: if you say `except Exception:` I believe pyflakes wont yell at you.

**Comment by Hellowlol** [comment]
Btw, is the plexapi username password and token in the test computers env? I would like to swap to using env as the current solution uses the config. And i got my own server in the config.. It make local testing painfull.

File: tests/test_video.py [hunk 2]
```diff
@@ -61,6 +60,15 @@ def test_video_Movie_isPartialObject(movie):
     assert movie.isPartialObject()
 
 
+def test_video_Movie_delete_part(movie, mocker):
+    # we need to reload this as there is a bug in part.delete
+    # See https://github.com/pkkid/python-plexapi/issues/201
+    m = movie.reload()
+    for part in m.iterParts():
```
Comment (Hellowlol) [inline tests/test_video.py]: We should iterate over media

**Comment by Hellowlol** [comment]
I'll go ahead and merge this. It's gonna be a while until I have time to complete the rest

**Comment by pkkid** [comment]
:+1:

**Pull request merged**